### PR TITLE
Suggest running clippy to aid auditing

### PR DESCRIPTION
**Pull request by Shnatsel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,9 +27,12 @@ Our process is as follows:
     [cargo geiger](https://github.com/anderejd/cargo-geiger) command can also
     help here.
   * Once you know where the `unsafe` blocks are it gets harder: you have to
-    carefully determine if the `unsafe` is being used appropriately. If you
-    don't know that's okay, post the questionable block in a comment in the
-    tracking issue here  and someone else can have a look too, or ask in
+    carefully determine if the `unsafe` is being used appropriately. We've been
+    [requesting Clippy lints](https://github.com/rust-secure-code/safety-dance/issues/21)
+    for known antipatterns, so running `cargo +nighly clippy` is a good
```
Comment (alex) [inline README.md]:
```suggestion
    for known antipatterns, so running `cargo +nightly clippy` is a good
```

File: README.md [hunk 2]
```diff
@@ -27,9 +27,12 @@ Our process is as follows:
...
+    starting point. If you don't know if a certain unsafe block is okay,
```
Comment (alex) [inline README.md]:
```suggestion
    starting point. If you don't know if a certain `unsafe` block is okay,
```

**Comment by Shnatsel** [comment]
I think clicking "Apply suggestion" broke github :sweat_smile:

**Comment by Shnatsel** [comment]
Good catch, thanks! Applied changes and pushed to master to work around Github having a bad day.

**Pull request closed without merging**